### PR TITLE
TAC-3296 support account requested

### PR DESCRIPTION
**Pull request by rasmuslager**
Account requested events v 0.7.0
* Account requested 
* accountRequestSucceeded 
* accountRequestFailed

### CODE REVIEW

File: example-app/src/main/java/com/getdreams/example/ExampleApp.kt [hunk 1]
```diff
@@ -13,6 +13,6 @@ import com.getdreams.Dreams
 class ExampleApp : Application() {
     override fun onCreate() {
         super.onCreate()
-        Dreams.configure(Dreams.Configuration(clientId = "clientId", baseUrl = "https://getdreams.io"))
+        Dreams.configure(Dreams.Configuration(clientId = "clientId", baseUrl = "http://acme.dreams.test:3000"))
```
Comment (abdulbaig1) [inline example-app/src/main/java/com/getdreams/example/ExampleApp.kt:16]: what should be our new URL here now? 🤔

Comment (abdulbaig1) [inline example-app/src/main/java/com/getdreams/example/ExampleApp.kt]: @rasmuslager what should be our new URL here now? 🤔

Comment (rasmuslager) [reply example-app/src/main/java/com/getdreams/example/ExampleApp.kt]:
```suggestion
        Dreams.configure(Dreams.Configuration(clientId = "clientId", baseUrl = "https://sales.dreams-demo.io"))
```

File: sdk/src/main/java/com/getdreams/connections/webview/RequestInterface.kt [hunk 2]
```diff
@@ -82,6 +82,7 @@ interface RequestInterface {
      * @param requestId The request id of the event that triggered the account provisioning.
      */
     fun accountProvisionInitiated(requestId: String)
+    fun accountRequestCompleted(requestData: String, success: Boolean)
```
Comment (rasmuslager) [inline sdk/src/main/java/com/getdreams/connections/webview/RequestInterface.kt]: Doc above might be missing

**Review by abdulbaig1 — APPROVED** [review]
as discussed and tested!

**Pull request merged**